typescript/packages/plugins/ionic/src/ionic.config.ts

```diff
@@ -0,0 +1,90 @@
+            "wrsETH": { address: "0x49950319aBE7CE5c3A6C90698381b45989C99b46", decimals: 18 },
+            "weETH": { address: "0xA0D844742B4abbbc43d8931a6Edb00C56325aA18", decimals: 18 },
+            "dMBTC": { address: "0x5158ae44C1351682B3DC046541Edf84BF28c8ca4", decimals: 8 },
+        },
+        PoolDirectory: "0x39C353Cf9041CcF467A04d0e78B63d961E81458a", // Mode PoolDirectory
+        PoolLens: "0x41576595C989b4958b974117B3755547f94dd380",      // Mode PoolLens
+    },
+// Base Chain
+    8453: {
+        pools: {
+            "1": "0x05c9C6417F246600f8f5f49fcA9Ee991bfF73D13",
+        },
+        assets: {
+            "ezETH": { address: "0x079f84161642D81aaFb67966123C9949F9284bf5", decimals: 18 },
+            "wstETH": { address: "0x9D62e30c6cB7964C99314DCf5F847e36Fcb29ca9", decimals: 18 },
+            "cbETH": { address: "0x9c201024A62466F9157b2dAaDda9326207ADDd29", decimals: 18 },
+            "AERO": { address: "0x014e08F05ac11BB532BE62774A4C548368f59779", decimals: 18 },
+            "USDC": { address: "0xa900A17a49Bc4D442bA7F72c39FA2108865671f0", decimals: 6 },
+            "WETH": { address: "0x49420311B518f3d0c94e897592014de53831cfA3", decimals: 18 },
+            "weETH": { address: "0x84341B650598002d427570298564d6701733c805", decimals: 18 },
+            "eUSD": { address: "0x9c2A4f9c5471fd36bE3BBd8437A33935107215A1", decimals: 18 },
+            "bsdETH": { address: "0x3D9669DE9E3E98DB41A1CbF6dC23446109945E3C", decimals: 18 },
+            "hyUSD": { address: "0x751911bDa88eFcF412326ABE649B7A3b28c4dEDe", decimals: 18 },
+            "RSR": { address: "0xfc6b82668E10AFF62f208C492fc95ef1fa9C0426", decimals: 18 },
+            "wsuperOETHb": { address: "0xC462eb5587062e2f2391990b8609D2428d8Cf598", decimals: 18 },
+            "wUSDM": { address: "0xe30965Acd0Ee1CE2e0Cd0AcBFB3596bD6fC78A51", decimals: 18 },
+            "cbBTC": { address: "0x1De166df671AE6DB4C4C98903df88E8007593748", decimals: 8 },
+            "EURC": { address: "0x0E5A87047F871050c0D713321Deb0F008a41C495", decimals: 18 },
+            "OGN": { address: "0xE00B2B2ca7ac347bc7Ca82fE5CfF0f76222FF375", decimals: 18 },
+            "USD+": { address: "0x74109171033F662D5b898A7a2FcAB2f1EF80c201", decimals: 18 },
+            "USDz": { address: "0xa4442b665d4c6DBC6ea43137B336e3089f05626C", decimals: 18 },
+            "wUSD+": { address: "0xF1bbECD6aCF648540eb79588Df692c6b2F0fbc09", decimals: 18 },
+            "sUSDz": { address: "0xf64bfd19DdCB2Bb54e6f976a233d0A9400ed84eA", decimals: 18 },
+            "uSOL": { address: "0xbd06905590b6E1b6Ac979Fc477A0AebB58d52371", decimals: 9 },
+            "uSUI": { address: "0xAa255Cf8e294BD7fcAB21897C0791e50C99BAc69", decimals: 6 },
+            "fBOMB": { address: "0xd333681242F376f9005d1208ff946C3EE73eD659", decimals: 18 },
+            "KLIMA": { address: "0x600D660440f15EeADbC3fc1403375e04b318F07e", decimals: 18 },
+        },
+        PoolDirectory: "0x4b49784CB7fB959cA78c88a9487197fC0497739f", // Base PoolDirectory
```

Comment:
Does not exist:

https://basescan.org/address/0x4b49784CB7fB959cA78c88a9487197fC0497739f